src/core/surface/alarm.c

```diff
@@ -63,9 +63,9 @@ grpc_alarm *grpc_alarm_create(grpc_completion_queue *cq, gpr_timespec deadline,
   alarm->cq = cq;
   alarm->tag = tag;
 
+  grpc_cq_begin_op(cq, tag);
```

Comment:
I'm curious, why the line move? I guess because there's potentially less interference with the time interval moving this unrelated op before we start the timer?